Framework/Geometry/src/RandomPoint.cpp

```diff
@@ -100,8 +109,8 @@ Kernel::V3D inHollowCylinder(const detail::ShapeInfo &shapeInfo,
   const double c2 = geometry.radius * geometry.radius;
   const double r{std::sqrt(c1 + (c2 - c1) * r2)};
   const double z{geometry.height * r3};
+  const Kernel::V3D basis1{geometry.axis};
```

Comment:
`basis1` here is just an alias to `geometry.axis`. To me, the variable does not make much sense anymore as the basis vector stuff has been moved to `localPointInCylinder()`. I think `basis` could now be removed and `geometry.axis` used directly.